diff --git a/src/classes.h b/src/classes.h
index 5d98742..8c14e7d 100644
--- a/src/classes.h
+++ b/src/classes.h
@@ -1,3 +1,9 @@
+#include "HiggsAnalysis/CombinedLimit/interface/RooACProcessScaling_1D.h"
+#include "HiggsAnalysis/CombinedLimit/interface/RooACProcessScaling_2D.h"
+#include "HiggsAnalysis/CombinedLimit/interface/RooACProcessScaling_3D.h"
+#include "HiggsAnalysis/CombinedLimit/interface/RooACSemiAnalyticPdf_1D.h"
+#include "HiggsAnalysis/CombinedLimit/interface/RooACSemiAnalyticPdf_2D.h"
+#include "HiggsAnalysis/CombinedLimit/interface/RooACSemiAnalyticPdf_3D.h"
 #include "HiggsAnalysis/CombinedLimit/interface/TestProposal.h"
 #include "HiggsAnalysis/CombinedLimit/interface/DebugProposal.h"
 #include "HiggsAnalysis/CombinedLimit/interface/VerticalInterpPdf.h"
diff --git a/src/classes_def.xml b/src/classes_def.xml
index 44aa8be..f403244 100644
--- a/src/classes_def.xml
+++ b/src/classes_def.xml
@@ -207,4 +207,10 @@
 	<class name="cmsmath::SequentialMinimizer"  transient="true" />
 	<class name="rVrFLikelihood"  transient="true" />
         <class name="TestProposal"  transient="true" />
+	<class name="RooACProcessScaling_1D"  transient="true" />
+	<class name="RooACProcessScaling_2D"  transient="true" />
+	<class name="RooACProcessScaling_3D"  transient="true" />
+	<class name="RooACSemiAnalyticPdf_1D"  transient="true" />
+	<class name="RooACSemiAnalyticPdf_2D"  transient="true" />
+	<class name="RooACSemiAnalyticPdf_3D"  transient="true" />
 </lcgdict>
